buildkite/bazelci.py

```diff
@@ -1631,7 +1631,10 @@ def download_bazelci_agent(dest_dir, platform, version):
     if platform == "windows":
         postfix = "x86_64-pc-windows-msvc.exe"
     elif platform == "macos":
-        postfix = "x86_64-apple-darwin"
+        if platform.machine() == 'arm64':
```

Comment:
we have to rename the platform module since it conflicts with the variable. I'll do it in my change

Reply:
ha, yes, sorry I didn't test it.